components/internal/modals/EditPermitHolderInformationModal.tsx

```diff
@@ -0,0 +1,186 @@
+import {
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalFooter,
+  ModalBody,
+  Button,
+  FormControl,
+  FormLabel,
+  Input,
+  useDisclosure,
+  Text,
+  Stack,
+  FormHelperText,
+  Box,
+  Divider,
+} from '@chakra-ui/react'; // Chakra UI
+import { useState, SyntheticEvent } from 'react'; // React
+
+export default function EditPermitHolderInformationModal() {
+  const { isOpen, onClose, onOpen } = useDisclosure();
+
+  // Personal information state
+  const [firstName, setFirstName] = useState('');
+  const [lastName, setLastName] = useState('');
+
+  // Contact information state
+  const [email, setEmail] = useState('');
+  const [phoneNumber, setPhoneNumber] = useState('');
+
+  // Home address information state
+  const [addressLine1, setAddressLine1] = useState('');
+  const [addressLine2, setAddressLine2] = useState('');
+  const [city, setCity] = useState('');
+  const [postalCode, setPostalCode] = useState('');
+
+  //   TODO: Add error states for each field (post-mvp)
+
+  const handleSubmit = (event: SyntheticEvent) => {
+    event.preventDefault();
+    // TODO: Will be addressed in API hookup
+  };
+
+  return (
+    <>
+      {/* Button will be removed before merging */}
+      <Button mt={3} onClick={onOpen}>
+        Open
+      </Button>
+
+      <Modal onClose={onClose} isOpen={isOpen} scrollBehavior="inside" size="3xl">
+        <ModalOverlay />
+        <form onSubmit={handleSubmit}>
+          <ModalContent paddingX="36px">
+            <ModalHeader
+              textStyle="display-medium-bold"
+              paddingBottom="12px"
+              paddingTop="24px"
+              paddingX="4px"
+            >
+              <Text textStyle="display-medium-bold">{'Edit Permit Holder Information'}</Text>
+            </ModalHeader>
+            <ModalBody paddingY="20px" paddingX="4px">
+              {/* Personal Information Section */}
+              <Box paddingBottom="32px">
+                <Text textStyle="heading" paddingBottom="24px">
+                  {'Personal Information'}
+                </Text>
+                <Stack direction="row" spacing="20px">
+                  <FormControl isRequired>
```

Comment:
Are you referring to the red highlighting and error message that appears under the form field (first image)?
![image](https://user-images.githubusercontent.com/51224641/127582448-90416f40-26d7-460d-ac2f-c68c5889f9d2.png)

If so that will be addressed post-mvp because of time constraints and in the meantime, Chakra's shows a small pop up under the required field (image below). I confirmed this with Oustan and the designers and left a small note in the PR description
![image](https://user-images.githubusercontent.com/51224641/127582666-9a2d0da1-93c1-4c91-826c-96f60fdd4d5d.png)

Reply:
Yep that's what I was referring to, thanks for clarifying!